x/dualstaking/client/cli/tx_delegate.go

```diff
@@ -50,6 +52,11 @@ func CmdDelegate() *cobra.Command {
 				argChainID,
 				argAmount,
 			)
+
+			if msg.Amount.Denom != commontypes.TokenDenom {
```

Comment:
token denom validation needs to be in the message server, not in the CLI.
we dont want to use the commontypes.tokenDenom

Reply:
this is also the common practice in cosmos

Reply:
Fixed in fcee22d6fcfd4ffacef8ec0181f5ee04ed1495c7